fastn-core/src/package/package_doc.rs

```diff
@@ -571,20 +571,34 @@ pub(crate) async fn read_ftd_2023(
             format!("{js_ftd_script}\n{js_document_script}").as_str(),
         )
     } else {
-        let ssr_body = if config.request.is_bot() {
-            fastn_js::ssr_with_js_string(
+        let (ssr_body, meta_tags) = if config.request.is_bot() {
+            let ssr_res = fastn_js::ssr_with_js_string(
```

Comment:
I would prefer if this function returned a string tuple instead of vec, as these checks we have performed as internal details of this function.

Reply:
Done in